<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>401</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>401</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>qtFFmpeg</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineInput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelInput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>INPUT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineOutput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>OUTPUT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonInput">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonOutput">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textCmd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>361</width>
       <height>91</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonRun">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>361</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonFFmpeg">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineFFMpeg">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelFFmpeg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ffmpeg</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>226</y>
       <width>361</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelFFmpeg_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Format</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>261</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>WAV (PCM s16le 44100 stereo)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MP3 High (cbr 320k stereo)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MP3 Medium (cbr 192k stereo)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MP3 Low (cbr 96k mono)</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
